PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java

```diff
@@ -69,19 +69,19 @@
     // Shared Preferences
     private SharedPreferences mUserPreferences;
     private ProgressDialog mProgressDialog;
-    /** Parent Activity **/
+    // Parent Activity *
```

Comment:
I'm pretty sure  // is better.  /** **/ just deems too messy. I've been changing them to //